src/components/Output.js

```diff
@@ -138,7 +165,10 @@ class Output extends Component<Props, State> {
           onLoad={this.onEditorLoad}
           readOnly
           {...aceStyleProps}
-          markers={this.state.showPython ? this.getMarkers() : []}
+          markers={
+            this.state.showPython ? this.getMarkers() : this.getTraceMarkers()
+          }
+          style={pythonErrorCall ? { color: 'red' } : {}}
```

Comment:
This also needs to work on coconut errors so maybe change to `(pythonErrorCall || coconutErrorCall)`. This line seems a bit messy.

Reply:
I think the `coconutErrorCall` refers to just an error on the editor side.

Reply:
But we should also color the output red
<img width="774" alt="screen shot 2018-04-10 at 8 28 29 pm" src="https://user-images.githubusercontent.com/8571671/38594945-c5afaa0c-3cfd-11e8-9a95-d1b6e504e4b7.png">